undebt/cmd/main.py

```diff
@@ -43,7 +43,13 @@ def _load_text(path):
 
 
 @_exit_fail_upon_error
-def _write_result_text(result_text, path):
+def _write_result_text(result_text, path, dry_run):
+    if dry_run:
+        if path:
+            sys.stdout.write('>>> {}\n'.format(path))
```

Comment:
I'd probably put this on stderr, that way one *could* pipe this to be meaningful, maybe?  shrugs not a big deal